core/src/main/java/tech/pegasys/web3signer/core/metrics/SlashingProtectionMetrics.java

```diff
@@ -34,6 +36,12 @@ public SlashingProtectionMetrics(final MetricsSystem metricsSystem) {
             ETH2_SLASHING_PROTECTION,
             "prevented_signings",
             "The number of prevented signings due to violation of slashing conditions.");
+
+    this.databaseTimer =
+        metricsSystem.createTimer(
```

Comment:
Would be good to create a timer with a label so we can database queries broken down into attestation and block duration times.